backend/question_service2/src/server.js

```diff
@@ -0,0 +1,32 @@
+// This module initialises the QuestionManager module, by setting up the MongoDB Server + its associated RESTAPI interface.
+
+// Load Env Variables
+if (process.env.NODE_ENV != 'production') {
+    require('dotenv').config();
+}
+
+// Import Dependencies
+const express = require('express');
+const bodyparser = require('body-parser'); // Middleware
+const questionmanager = require('./questionmanager');
+
+// Create an express app
+const app = express();
+
+// Configure express app
+app.use(bodyparser.json());
+app.use(bodyparser.urlencoded({ extended: true }));
+
+app.post('/questions/create', questionmanager.createQuestion); //CREATE
+app.get('/questions/getall', questionmanager.getQuestions); // READ
+app.get('/questions/get/:qn_num', questionmanager.getQuestionByQnNum);
+app.post('/questions/update/:qn_num', questionmanager.updateQuestion); //UPDATE
+app.post('/questions/delete/:qn_num', questionmanager.deleteQuestion); //DELETE')
```

Comment:
Good job in refactoring into `post` and `get`.

Reply:
Thanks!